src/it/package-exclude-by-group-it/invoker.properties

```diff
@@ -0,0 +1,18 @@
+#
+#
+#   Copyright (c) 2016 Red Hat, Inc.
+#
+#   Red Hat licenses this file to you under the Apache License, version
+#   2.0 (the "License"); you may not use this file except in compliance
+#   with the License.  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+#   implied.  See the License for the specific language governing
+#   permissions and limitations under the License.
+#
+invoker.mavenOpts=-Xdebug -Xnoagent -Xrunjdwp:transport=dt_socket,server=y,suspend=y,address=9001
```

Comment:
Not sure you want `suspend=y`. The [CI build](https://fabric8-ci.fusesource.com/job/vertx-maven-plugin-pullreq/70/console) has been waiting 17 hours for someone to connect a debugger :wink: